test/Microsoft.Health.Fhir.Shared.Tests.Integration/Features/Operations/SqlServerFhirOperationDataStoreTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Numerics;
+using System.Threading.Tasks;
+using Microsoft.Health.Fhir.Tests.Common.FixtureParameters;
+using Microsoft.Health.Fhir.Tests.Integration.Persistence;
+using Xunit;
+
+namespace Microsoft.Health.Fhir.Tests.Integration.Features.Operations
+{
+    [Collection(FhirOperationTestConstants.FhirOperationTests)]
+    [FhirStorageTestsFixtureArgumentSets(DataStore.SqlServer)]
+    public class SqlServerFhirOperationDataStoreTests : IClassFixture<FhirStorageTestsFixture>
```

Comment:
This isn't really specific to operations, is it?

Reply:
Good point. Renamed the file and moved it to a different folder in 7ed0da5.